### PR TITLE
Create 002.md

### PR DESCRIPTION
**Pull request by chris-short**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: archive/002.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* The MCS-Controller release version is [v0.2.2](https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/releases/tag/v0.2.2) with GA expected in H2 CY2022
+* Includes a tutorial that walks through how setup a multi-cluster service with MCS
+
+## Containers from the Couch
+
+[Using Kubecost to monitor your Kubernetes and infrastructure spend](https://youtu.be/2IJntyJy6vY)  
+We look at how [Kubecost](https://www.kubecost.com/) can be used to track your infrastructure spending and save you money!
+
+[Kubernetes Virtual clusters with Loft Labs](https://youtu.be/a8fIyUd9438)
+We'll look at benefits and use cases for Kubernetes virtual clusters using [Loft](https://loft.sh/).
```
Comment (rothgar) [inline archive/002.md:21]: You could mention that vclusters are completely free at vcluster.com and you'll (hopefully) soon be able to use EKS distro for virtual clusters via https://github.com/loft-sh/vcluster/pull/320

File: archive/002.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* AWS Cloud Map MCS is an implementation of the [MCS API](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api#kep-1645-multi-cluster-services-api), allowing you discover and access services outside a cluster
+* 2 primary scenarios: **Different services each deployed to separate clusters** and **Single service deployed to multiple clusters**
+* The MCS-Controller is responsible for “syncing” services across multiple clusters with Cloud Map serving as the centralized service registry
+* The MCS-Controller release version is [v0.2.2](https://github.com/aws/aws-cloud-map-mcs-controller-for-k8s/releases/tag/v0.2.2) with GA expected in H2 CY2022
+* Includes a tutorial that walks through how setup a multi-cluster service with MCS
+
+## Containers from the Couch
+
+[Using Kubecost to monitor your Kubernetes and infrastructure spend](https://youtu.be/2IJntyJy6vY)  
+We look at how [Kubecost](https://www.kubecost.com/) can be used to track your infrastructure spending and save you money!
```
Comment (rothgar) [inline archive/002.md:18]: You should also link to the kubectl plugin https://github.com/kubecost/kubectl-cost

Comment (chris-short) [reply archive/002.md]: Done.

File: archive/002.md (hunk 1 above)
Comment (chris-short) [reply archive/002.md]: Done.

**Pull request merged**